serialization/src/main/scala/sbt/serialization/CustomPicklerUnpickler.scala

```diff
@@ -1,18 +1,23 @@
 package sbt.serialization
 
-import sbt.serialization.pickler.{
+// TODO - Why are type aliases not workign?
```

Comment:
Since you can access members of package object serialization only after it's constructed, and to construct the object named `package`, you need to compile its parent trait `CustomPicklers`?